pkg/api/script.go

```diff
@@ -3,6 +3,9 @@ package api
 // Script defines script names used by STI process.
 type Script string
 
+// Location defines script locations used by STI process.
+type Location string
```

Comment:
nuke this ;-) I think this is really a bad practice to mask the string into special type....